### ISSUE TITLE
[Fixed] Gerald not working due to fire

### ISSUE THREAD

**Issue by jackie1050** [DoESLiverpool/somebody-should#290]
There was a fairly large fire while students were laser cutting detailed parts in cardboard. The air cooling part does not appear to be functioning correctly (and a bit burnt). Patrick said this could have been the original cause of the fire.
Lessons 
- check under fire blanket, and under bed, to make sure fire has gone out
- clear out under bed more regularly

Switched off at key - may need new part

**Comment by DefProc** [comment]
It looks like the only damage to the machine was the air supply pipe to the head, presumably from being above the cut material that was on fire for an amount of time. I've fixed it for now with a narrower tube, but I have some 5mm ID PVC hose, with a pair on straight fittings on order. 

My initial investigation suggests that the cause of the fire was cutting multiple sheets of cardboard at the same time ­— a practice I have recommended against because of the increased risk of fire, and it's inefficiency compared to multiple single sheets. It's possible that multiple overlapping cuts exacerbated the problem by the continued heating of a small region, and has then caused the base material to set on fire. 

While the user did seek help from another workshop member at the time of the fire, both had been inducted on the laser cutter, and followed the fire safety procedure. As the users felt that the fire was large enough that the fire blanket was used. Unfortunately, the fire blanket was draped over the laser's x-axis bar, and not directly on the the surface of the burning material, so the fire was not immediately smothered. 

Recomendations: 

@DoESsean can we add an agenda item for the next organiser's meeting to review the laser cutter fire procedures, to look at if: 

 - the current fire blankets are still suitable, 
 - or another extinguisher should be used in the future, 
 - or if revised information during the laser induction is sufficient. 

Revise the laser information wiki pages to specifically note the problems of:

 - cutting multiple sheets at once, 
 - not cutting lines more than once where possible.
